src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
```

Comment:
Prefer `if` with `return` over `if else` to simplify the conditions. After the `if` condition with a `return`, the `else` is unnecessary because the code after it won't be executed if the condition is true. This helps to reduce nesting and makes the code more readable. Refer to checklist items 4 and 5.